exporter/fairshare.go

```diff
@@ -0,0 +1,156 @@
+package exporter
```

Comment:
Unfortunately, for rivos OSS projects we need to add the following header to every file for licensing reasons. 


```suggestion
// SPDX-FileCopyrightText: 2023 Rivos Inc.
//
// SPDX-License-Identifier: Apache-2.0

package exporter
```